### PR TITLE
remove missing claim logging

### PR DESCRIPTION
**Pull request by jbspeakr**
see #84

### CODE REVIEW

File: handlers/tokeninfo/jwt/tokeninfo.go [hunk 1]
```diff
@@ -174,12 +174,11 @@ func getClaim(t *jwt.Token, claim string) (interface{}, bool) {
 	if claims, ok := t.Claims.(jwt.MapClaims); ok {
 		c, has := claims[claim]
 		if !has {
-			log.Printf("Missing claim %q for token %v", claim, maskToken(t.Raw))
 			return "", false
 		}
 		return c, true
 	}
-	log.Printf("Missing claim %q for token", claim)
+	log.Printf("Missing claim %q for token %v", claim, maskToken(t.Raw))
```
Comment (vroldanbet) [inline handlers/tokeninfo/jwt/tokeninfo.go]: do we need this one either?

Comment (jbspeakr) [inline handlers/tokeninfo/jwt/tokeninfo.go]: I think logging unexpected claims in incoming tokens makes sense.

Comment (vroldanbet) [inline handlers/tokeninfo/jwt/tokeninfo.go]: but is this executed when the jwt claims cannot be casted to MapClaims? how is this related to unexpected claims, am I missing something?

Comment (jbspeakr) [inline handlers/tokeninfo/jwt/tokeninfo.go]: I am not the right guy to ask for the requirements that lead to the implementation of this line back in the days. I just corrected it. If you say it can be safely removed I'll do so.

Comment (vroldanbet) [inline handlers/tokeninfo/jwt/tokeninfo.go]: not that much of a "if you say so". The function returns an boolean indicating if the claim is there or not. Clients can decide to log the message if necessary. Thanks for the PR!

**Comment by vroldanbet** [comment]
:+1:

**Comment by smirnov** [comment]
👍
